.builders/scripts/build_wheels.py

```diff
@@ -112,10 +112,14 @@ def main():
             str(python_path), '-m', 'pip', 'wheel',
             '-r', str(MOUNT_DIR / 'requirements.in'),
             '--wheel-dir', str(staged_wheel_dir),
-            '--extra-index-url', CUSTOM_EXTERNAL_INDEX,
+            # Temporarily removing extra index urls. See below.
+            # '--extra-index-url', CUSTOM_EXTERNAL_INDEX,
         ]
-        if args.use_built_index:
-            command_args.extend(['--extra-index-url', CUSTOM_BUILT_INDEX])
+        # Temporarily disable extra index urls. There are broken wheels in the gcloud bucket
```

Comment:
I see. Is there a ticket or other link we could have here to track the removal of the temporary countermeasure?

Reply:
We need to rethink the process because currently the wheels would not build after we have removed the tests we do not want shipped with the agnet. We will do that as part of the test removal work (#20653) which correspond to the issue AI-5519.

Thanks!

Reply:
Thank you!